bigquery/google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Cursor for the Google BigQuery DB-API."""
+
+import collections
+import uuid
+
+import six
+
+from google.cloud.bigquery.dbapi import _helpers
+from google.cloud.bigquery.dbapi import exceptions
+
+
+_ARRAYSIZE_DEFAULT = 20
+
+
+class Cursor(object):
+    """DB-API Cursor to Google BigQuery.
+
+    :type: :class:`~google.cloud.bigquery.dbapi.Connection`
+    :param connection: A DB-API connection to Google BigQuery.
+    """
+    def __init__(self, connection):
+        self.connection = connection
+        self.description = None
+        self.rowcount = -1
```

Comment:
Can you add a comment here explaining "Per PEP 239: The attribute is -1 in case no .execute*() has been performed on the cursor or the rowcount of the last operation is cannot be determined by the interface."